completers/git_completer/cmd/root.go

```diff
@@ -57,4 +62,60 @@ func init() {
 		}
 		return action
 	})
+
+	addAliasCompletion(rootCmd)
+}
+
+func addAliasCompletion(cmd *cobra.Command) {
```

Comment:
I mostly based this logic off of https://github.com/rsteube/carapace-bin/blob/ad97d14b023eb926384e3c778f03ad069aa2c0cc/completers/gh_completer/cmd/root.go#L29-L47